alis.sh

```diff
@@ -1475,6 +1475,12 @@ function desktop_environment() {
             ;;
         "openbox" )
             desktop_environment_openbox
+                ;;
+        "dusk" )
+                desktop_environment_dusk
+                ;;
```

Comment:
Set same indentation for the lines as other DE

Reply:
will you do that or I?